setup.sh

```diff
@@ -7,7 +7,12 @@ mkdir -p "$INSTALL_DIR"
 git clone --depth 1 https://gerrit.wikimedia.org/r/mediawiki/core "$INSTALL_DIR"/mediawiki
 git clone --depth 1 https://gerrit.wikimedia.org/r/mediawiki/skins/Vector "$INSTALL_DIR"/mediawiki/skins/Vector
 
-cd "$INSTALL_DIR"/mediawiki
+
+DOCKER_MW_PATH="$INSTALL_DIR"/mediawiki
```

Comment:
Might make sense to set this earlier, and then use it in the git clone lines too?

Reply:
Done!